CHANGELOG.md

```diff
@@ -89,7 +89,6 @@ All notable changes to the Aptos TypeScript SDK will be captured in this file. T
 - Includes the address in the `AbstractKeylessAccount` serialization to prevent information loss for key rotated accounts.
 - [`Breaking`] Deprecate `serializeOptionStr` and `deserializeOptionStr` in favor of `serializeOption` and `deserializeOption`.
 - [`Breaking`] Renames `KeylessConfiguration.verficationKey` to `verificationKey`
-- Add a new `scriptComposer` api in transactionSubmission api to allower SDK callers to invoke multiple Move functions inside a same transaction and compose the calls dynamically.
```

Comment:
Please revert this change. And add a new change to the changelog saying "Remove scriptComposer api due to increase in the sdk bundle size"